watertap/costing/watertap_costing_package.py

```diff
@@ -113,6 +114,11 @@ def build_global_params(self):
             doc="Membrane replacement factor [fraction of membrane replaced/year]",
             units=pyo.units.year**-1,
         )
+        self.factor_uv_replacement = pyo.Var(
+            initialize=0.3993,
+            doc="UV replacement factor [fraction of uv replaced/year]",

```

Comment:
Is this restricted to lamp replacement?

Reply:
I sum up the consumable costs of lamps, sleeves, ballasts and sensors, so it accounts for all of them.

Reply:
```suggestion
            doc="UV replacement factor accounting for lamps, sleeves, ballasts and sensors [fraction of uv replaced/year]",

```